app/explorer/Explorer.jsx

```diff
@@ -68,6 +69,15 @@ class Explorer extends React.Component {
   }
 
   onLayerToggle = (layerId, wasEnabled) => {
+    ReactGA4.gtag('event', 'toggle-layer', {
```

Comment:
I thought we were going to get away from tracking individual events in GA? (in favor of using FS for that sort of thing)